gateway/api_definition.go

```diff
@@ -441,16 +471,16 @@ func (f *nestedApiDefinitionList) filter(enabled bool, tags ...string) []nestedA
 // FromDashboardService will connect and download ApiDefintions from a Tyk Dashboard instance.
 func (a APIDefinitionLoader) FromDashboardService(endpoint string) ([]*APISpec, error) {
 	// Get the definitions
-	log.Debug("Calling: ", endpoint)
 	newRequest, err := http.NewRequest("GET", endpoint, nil)
 	if err != nil {
-		log.Error("Failed to create request: ", err)
+		return nil, fmt.Errorf("failed to create request: %w", err)
 	}
 
 	gwConfig := a.Gw.GetConfig()
 
-	newRequest.Header.Set("authorization", gwConfig.NodeSecret)
 	log.Debug("Using: NodeID: ", a.Gw.GetNodeID())
+
+	newRequest.Header.Set("authorization", gwConfig.NodeSecret)
```

Comment:
we have `headers.Authorization` defined and can be used here

Reply:
Done.